uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -566,7 +566,7 @@ private static function mustUpdatePublicHolidayLeaveRequests($params) {
    */
   private static function enqueuePublicHolidayLeaveRequestUpdateTask() {
     $task = new CRM_Queue_Task(
-      ['CRM_HRLeaveAndAbsences_Queue_Task_UpdateAllFuturePublicHolidayLeaveRequests', 'run'],
+      ['CRM_HRLeaveAndAbsences_Queue_Task_UpdateAllPublicHolidayLeaveRequestsExceptThoseAlreadyCreatedInThePast', 'run'],
```

Comment:
I don't think we should be explicit about the name here either. A simple 'UpdateAllPublicHolidayLeaveRequests' should be enough